lib/rules/declaration-block-trailing-semicolon/index.js

```diff
@@ -49,12 +49,28 @@ const rule = function(expectation) {
         if (node.parent.raws.semicolon) {
           return;
         }
+
+        // auto-fix
+        if (context.fix) {
+          node.parent.raws.semicolon = true;
+          if (node.type === "atrule") {
```

Comment:
The purpose of this code isn't clear. What does it do?

Reply:
For such cases as `@import`. To make it clearer, I can change it to `node.type === "atrule" && node.name === "import`